grafana/README.md

```diff
@@ -0,0 +1,24 @@
+# Grafana Dashboard Import Guide
+
+## Overview
+
+This guide explains how to import and configure the provided Grafana dashboard.
+
+## Steps to Import
+
+1. **Download JSON**: Get the `dashboard.json` file.
+2. **Log in to Grafana**: Access your Grafana instance.
+3. **Import Dashboard**:
+    - Go to the "+" menu, select "Import".
+    - Upload the JSON file or paste its contents.
+4. **Configure Data Sources**: Map any required data sources to your existing ones.
+5. **Save**: Confirm and save the dashboard.
+
+## Notes
+
+- Ensure data sources are properly configured.
+- Customize variables, thresholds, and layouts as needed.
```

Comment:
Can you add more details how to configure amoro metric reporter？ Or put a href link to official website.

Reply:
Ok, I have added a link to official website.